packages/common/src/grpc-service.js

```diff
@@ -635,18 +642,27 @@ GrpcService.structToObj_ = function(struct) {
  * @param {?error} callback.err - An error getting an auth client.
  */
 GrpcService.prototype.getGrpcCredentials_ = function(callback) {
+  var self = this;
+
   this.authClient.getAuthClient(function(err, authClient) {
     if (err) {
       callback(err);
       return;
     }
 
-    var credentials = grpc.credentials.combineChannelCredentials(
+    var callCredentialObjects = [
       grpc.credentials.createSsl(),
       grpc.credentials.createFromGoogleCredential(authClient)
+    ];
+
+    callCredentialObjects = callCredentialObjects.concat(self.callCredentials);
```

Comment:
This is where the metadata CallCredential object [created here](https://github.com/GoogleCloudPlatform/google-cloud-node/pull/1546/files#diff-78d3ba232fe7028b5068f61d6d20a988R737) is combined with the `createSsl` and `createFromGoogleCredential` objects created above.